packages/big-design/src/components/Button/styled.tsx

```diff
@@ -30,6 +30,7 @@ export const StyledButton = styled.button<ButtonProps & MarginProps>`
   position: relative;
   text-align: center;
   text-decoration: none;
+  transition: background-color 150ms ease-out;
```

Comment:
For active, focus, and disabled state:
```suggestion
  transition: 150ms ease-out;
  transition-property: background-color, border-color, box-shadow, color;
```